jovian/jovian_nb_ext/main.js

```diff
@@ -344,20 +687,20 @@ define([
     );
 
     //toolbar button to remove the jovian extension
-    const remove_ext_action = {
-      icon: "fa-times",
+    const set_paras_ext_action = {
```

Comment:
Can you use `params` instead of `paras` in all instances

Reply:
Sure, I will change this, and also I will consider another to store parameters if there's any other better way.